grapher/lineCharts/LineChart.tsx

```diff
@@ -659,7 +659,7 @@ export class LineChart
         const { dualAxis } = this
         const { horizontalAxis, verticalAxis } = dualAxis
 
-        return this.series.map((series) => {
+        return this.series.reverse().map((series) => {
```

Comment:
One thing, after I just worked on https://github.com/owid/owid-grapher/pull/856:

You might want to make sure that we're not inadvertently modifying `this.series` in-place, or at least double-check that doing so doesn't introduce any bugs.

Reply:
Yeah good catch! Now marked as `readonly`, though a lot of other getters are unmarked!